<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsWidget</class>
 <widget class="QWidget" name="StatsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stats</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
